opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -556,24 +558,29 @@ def test_ended_span(self):
             end_time0 = root.end_time
 
             # call it a second time
-            root.end()
+            with self.assertLogs(level="WARNING"):
+                root.end()
             # end time shouldn't be changed
             self.assertEqual(end_time0, root.end_time)
 
-            root.set_attribute("component", "http")
+            with self.assertLogs(level="WARNING"):
+                root.set_attribute("component", "http")
             self.assertEqual(len(root.attributes), 0)
 
-            root.add_event("event1")
+            with self.assertLogs(level="WARNING"):
+                root.add_event("event1")
             self.assertEqual(len(root.events), 0)
 
-            root.update_name("xxx")
+            with self.assertLogs(level="WARNING"):
+                root.update_name("xxx")
             self.assertEqual(root.name, "root")
 
             new_status = trace_api.status.Status(
                 trace_api.status.StatusCanonicalCode.CANCELLED,
                 "Test description",
             )
-            root.set_status(new_status)
+            with self.assertLogs(level="WARNING"):
```

Comment:
Maybe it's better to use `logging.WARNING` etc. instead of string literal

Reply:
Yes, good point. Will change.